db/fixtures/miq_product_features.yml

```diff
@@ -3407,6 +3407,14 @@
       :description: Edit Tags of Middleware Deployments
       :feature_type: control
       :identifier: middleware_deployment_tag
+    - :name: Undeploy middleware deployment
+      :description: Undeploy Middleware deployment
+      :feature_type: admin
+      :identifier: middleware_deployment_undeploy
+    - :name: Redeploy middleware deployment
+      :description: Trigger redeploy operation for Middleware deployment
```

Comment:
Adapted the descriptions from the server (https://github.com/ammendonca/manageiq/blob/145809328858ab9a1678adc293f42b25ada220bf/db/fixtures/miq_product_features.yml#L3356-L3363), and the reasoning I understood was that redeploy (as with reload) involves a chained 2 step process, stopping and starting, where "trigger" made sense.
But maybe "Redeploy Middleware Deployment" is also good enough.